randovania/games/dread/presets/starter_preset.rdvpreset

```diff
@@ -3,7 +3,7 @@
     "base_preset_uuid": null,
     "name": "Starter Preset",
     "uuid": "407e97ed-decb-4889-8ab7-943f86ae48a4",
-    "description": "Basic preset.",
+    "description": "This preset is designed for those new to Metroid Dread or its randomizer. Every seed can be completed without utilizing any tricks or glitches.<br/><br/>Along with shuffling items, some changes have been made to improve the randomizer experience. These include starting with Pulse Radar, preventing some softlocks, and more. This preset requires collecting 3 Metroid DNA in order to access Raven Beak.",
```

Comment:
"without tricks" is kinda debatable?
Technically shinesparking is a trick, and we still have vanilla locations that require shinesparking being marked as trickless

Reply:
I don't think there's anything wrong with the wording here, but it can be worth mentioning that some of the Speed Booster puzzles aren't logical unless the corresponding trick is enabled.